qiskit/utils/backend_utils.py

```diff
@@ -102,6 +115,10 @@ def is_aer_provider(backend):
     return False
 
 
+@deprecate_func(
+    since="0.24.0",
+    additional_msg="This function was used by the deprecated QuantumInstance class. For code migration guidelines, visit https://qisk.it/qi_migration.",
```

Comment:
I will go through all of the QI-related messages tomorrow, because I see that they are not consistent.